<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <rosparam file="$(find mybot_pkg)/config/params.yaml"/>

    <node name="camera_calibration_node" pkg="camera_calibration" type="cameracalibrator.py" output="screen" > 
	    <param name="size" value="9x6" /> 
	    <param name="square" value="0.027" /> 
	    <param name="image" value="/usb_cam_node/image_raw" /> 
	    <param name="camera" value="/usb_cam_node" />
  	</node>
<!-- rosrun camera_calibration cameracalibrator.py --size 9x6 --square 0.027 image:=/usb_cam_node/image_raw camera:=/usb_cam_node -->
<!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen"> 
		<remap from="image" to="/usb_cam/image_raw"/> 
		<param name="autosize" value="true" /> 
	</node> -->
</launch>